### PR TITLE
Add more detailed instructions for the kvm driver

### PR DESCRIPTION
**Pull request by brk3**
Users must properly install and configure libvirt + qemu

Solves issues such as #278

### CODE REVIEW

**Comment by jimmidyson** [comment]
:+1:  thanks!

File: DRIVERS.md [hunk 1]
```diff
@@ -14,9 +14,20 @@ the host PATH:
...
+
+```
+$ sudo curl -L https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.7.0/docker-machine-driver-kvm -o /usr/local/bin/docker-machine-driver-kvm
+$ sudo chmod +x /usr/local/bin/docker-machine-driver-kvm
+
+# Install libvirt and qemu-kvm on your system, e.g.
+$ sudo apt install libvirt-bin qemu-kvm
+
+# Add yourself to the libvirtd group (may vary by linux distro) so you don't need to sudo
+$ sudo usermod -a -G libvirtd $(whoami)
```
Comment (dlorenc) [inline DRIVERS.md]: I think you'll still need to logout/back in to make this take effect, right?

Comment (brk3) [inline DRIVERS.md]: Ah good point

Comment (jimmidyson) [inline DRIVERS.md]: `newgrp libvirtd` should do it for the current shell.

Comment (brk3) [inline DRIVERS.md]: Done, thanks!

**Comment by dlorenc** [comment]
Awesome, thanks for the work!

**Pull request merged**